[2024-03-29T08:35:05.795+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: demo_dag_target.run_this manual__2024-03-29T08:35:05.090646+00:00 [queued]>
[2024-03-29T08:35:05.804+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: demo_dag_target.run_this manual__2024-03-29T08:35:05.090646+00:00 [queued]>
[2024-03-29T08:35:05.804+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-03-29T08:35:05.817+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): run_this> on 2024-03-29 08:35:05.090646+00:00
[2024-03-29T08:35:05.823+0000] {standard_task_runner.py:60} INFO - Started process 1586 to run task
[2024-03-29T08:35:05.827+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'demo_dag_target', 'run_this', 'manual__2024-03-29T08:35:05.090646+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/demo_dag_target.py', '--cfg-path', '/tmp/tmpiajgme8h']
[2024-03-29T08:35:05.829+0000] {standard_task_runner.py:88} INFO - Job 4: Subtask run_this
[2024-03-29T08:35:05.843+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-29T08:35:05.883+0000] {task_command.py:423} INFO - Running <TaskInstance: demo_dag_target.run_this manual__2024-03-29T08:35:05.090646+00:00 [running]> on host 4cab8a94f129
[2024-03-29T08:35:05.972+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='domwatcharin' AIRFLOW_CTX_DAG_ID='demo_dag_target' AIRFLOW_CTX_TASK_ID='run_this' AIRFLOW_CTX_EXECUTION_DATE='2024-03-29T08:35:05.090646+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-29T08:35:05.090646+00:00'
[2024-03-29T08:35:05.973+0000] {demo_dag_target.py:10} INFO - Remotely received value: Hello World
[2024-03-29T08:35:05.974+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-03-29T08:35:05.989+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=demo_dag_target, task_id=run_this, execution_date=20240329T083505, start_date=20240329T083505, end_date=20240329T083505
[2024-03-29T08:35:06.049+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-29T08:35:06.066+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
